packages/__docs__/src/Search/index.tsx

```diff
@@ -70,18 +73,22 @@ class Search extends Component<SearchProps, SearchState> {
       })
     })
   }
+  componentDidUpdate(_prevProps: SearchProps, prevState: SearchState) {
+    if (this.state.announcement != prevState.announcement) {
+      this._debounced({ announcement: null })
+    }
```

Comment:
This is the actual fix. Now the SR alert disappears after 1 second. this is enough to read it, but when tabbing out from the search field it will no longer be in the beginning of the DOM